### DISCUSSION TITLE
wiki page about Pure Maps in the OSM wiki?

### DISCUSSION THREAD

**Discussion by stephan75** [rinigus/pure-maps#585] [General]
Hello @rinigus ,
I recently came across this app that seems to be OSM related in some features.

But so far I did **not** find an item for your software in the OSM wiki at https://wiki.openstreetmap.org ... right?

Would you appreciate such an own page in that wiki?

Maybe a template is available for such kind of apps that can be run on linux based devices?

**Comment by rinigus** [comment]
Indeed, it is missing from OSM WIKI. Pure Maps and OSM Scout Server are shown as missing on https://wiki.openstreetmap.org/wiki/Ubuntu_Touch , for example. Would be great if someone would take time and write it up there with the links. Pure Maps has also placeholder as a homepage (see https://github.com/rinigus/pure-maps/issues/400) - filling that is welcome as well.

**Comment by jonnius** [comment]
I just updated the Pure Maps and OSM Scout Server entries in the [table](https://wiki.openstreetmap.org/wiki/Ubuntu_Touch). Does anyone know what the "Monitor" column is about?

**Reply by rinigus** [reply]
No idea. Could it be that it keeps an eye on GPS?

**Reply by jonnius** [reply]
OSM Scout Server should not have it then. I asked in the [discussion section](https://wiki.openstreetmap.org/wiki/Talk:Ubuntu_Touch).

**Reply by rinigus** [reply]
Indeed

**Comment by jonnius** [comment]
I would be happy to see someone write a wiki page about Pure Maps (and OSM Scout Server) on the OSM wiki. German speakers can use the [ubuntuusers page](https://wiki.ubuntuusers.de/Ubuntu_Touch/Pure_Maps/) as template.
